pkgs/development/python-modules/pwntools/python3.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub
+, Mako, packaging, pysocks, pygments, ROPGadget
+, capstone, paramiko, pip, psutil
+, pyelftools, pyserial, dateutil
+, requests, sortedcontainers, tox, unicorn
+, intervaltree }:
+
+
+buildPythonPackage rec {
+  version = "dev3";
```

Comment:
dev3 is essentially their "master", i would use the "unsable-YYYY-MM-DD" for version, and use the commit hash for the rev in src.